locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,173 @@
+  { -brand-premium } subscribers.
+modal-heres-what-we-fixed-description-part-three = <b>In Progress</b> includes anything that we are currently
+  working on fixing. Removals typically take 7-14 days but the most
+  difficult sites could take longer. You may also start to see removals
+  happening within the same day.
+
+# About Exposure Types Modal
+
+modal-exposure-type-title = About exposure types
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be more than 1.
+modal-exposure-type-description = We search for you in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Here are the two types of exposures we find:
+modal-exposure-type-data-breach = <b>Data breach</b> means your information has been compromised in a breach and could be in the wrong hands.
+  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-premium }.
+modal-exposure-type-data-broker-part-one = <b>Info for sale</b> means a data broker site is publicly publishing and selling your personal info.
+  You’ll need to manually request removal from each site.
+modal-exposure-type-data-broker-part-two = If you’re a { -brand-premium } user, we automatically remove all profiles for you.
+  In both cases, removals typically take 7-14 days. Some can take longer, while others can happen within the hour.
+
+# About Exposure Statuses Modal
+
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+modal-exposure-status-description-premium = We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info.
+  Your exposures will have one of the following statuses:
+modal-exposure-status-in-progress = <b>In progress</b> means we are actively working on fixing the exposure for you. This is a { -brand-premium } feature.
+
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+# $data_broker_unresolved_num is the unresolved number of data brokers selling the user’s data.
+dashboard-exposures-area-description-premium = We found your information exposed { $exposures_unresolved_num } times over { $data_breach_unresolved_num } data breaches and { $data_broker_unresolved_num } data broker sites that are selling your personal info.
+
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+dashboard-exposures-breaches-scan-progress-description = {
```

Comment:
Remind me: are we going to keep this (and the string above) as-is for the time being?

Reply:
Yeah, I asked about it, but since it's US-only (and I think harder to break up into three parts), [it's being left as-is for now](https://mozilla.slack.com/archives/C0460QALW69/p1704304683210789?thread_ts=1704214729.664269&cid=C0460QALW69).